landlab/components/gravel_river_transporter/gravel_river_transporter.py

```diff
@@ -0,0 +1,484 @@
+import numpy as np
+from landlab import Component, HexModelGrid
+from landlab.grid.diagonals import DiagonalsMixIn
+from scipy.sparse.linalg import spsolve
+
+_ONE_SIXTH = 1.0 / 6.0
+
+
+def make_empty_matrix_and_rhs(grid):
+    from scipy.sparse import csc_matrix
+
+    mat = csc_matrix(
+        (grid.number_of_core_nodes, grid.number_of_core_nodes),
+    )
+    rhs = np.zeros(grid.number_of_core_nodes)
+    return mat, rhs
+
+
+def zero_out_matrix(grid, mat, rcvr, mat_id):
```

Comment:
If this function is not used by other classes, it can be defined as a static method within the class.

Reply:
Same as previous: could potentially be useful for multiple components in the future.